COVIDSafepassage/passsystem/views.py

```diff
@@ -18,6 +21,40 @@ def home(request):
     return render(request, 'passsystem/home.html')
 
 
+class SessionLoginApiView(APIView):
+    permission_classes = [IsAuthenticated]

```

Comment:
can we remove this line?

Reply:
sure. But I still kept it for later enhancements. We can have multiple permission classes like IsAdmin, IsSuperUser etc. Just to demonstrate the functionality